docs/user/reference/charm/index.md

```diff
@@ -1,97 +1,242 @@
 (charm)=
 # Charm
 
+> See also: {ref}`manage-charms`
+
 ```{toctree}
 :hidden:
-
-channel
-revision
-charm-taxonomy
 charm-development-best-practices
 charm-naming-guidelines
 charm-maturity
-charm-environment-variables
 ```
 
+In Juju, a **charm** is an operator -- software that wraps an {ref}`application <application>` and that contains all of the instructions necessary for deploying, configuring, scaling, integrating, etc., the applicationon any {ref}`Juju-supported cloud <list-of-supported-clouds`.
+
+
+## The charm store
+
+Charms are publicly available on [Charmhub](https://charmhub.io/).
```

Comment:
"in" rather that "on"? "from"?

Reply:
PS I know we also spoke about "charm taxonomy" being potentially too cryptic. I'm keeping it at that now (we have this language in other places too). I'll keep thinking, though.